autoprotocol/types/protocol.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
```

Comment:
this is the additional import you mentioned which is the tiebreaker in terms of which option you prefer?

Reply:
nvm. when you say consumers youre talking about ap writers, yeah? the import would be `from protocol.types import pickparams`

Reply:
an example of it is down in the test that uses autopick2